src/main/java/ulcrs/data/DataStore.java

```diff
@@ -37,8 +54,8 @@ private static DataStore getInstance() {
      * @param id - id of the tutor
      * @return Tutor - the Tutor with the specified id, if it exists. Otherwise, null.
      */
-    public static Tutor getTutor(int id) {
-        fetchIfRequired(getInstance().tutors);
+    public static Tutor getTutor(int id, String cookie) {

```

Comment:
String or the Cookie class? Both should work and for now we can use String. But if possible we should use Cookie in the future.

Reply:
I agree, for lack of time I just went with String since it worked, but this should be looked at further